EmbeddedSocialClient/sdk/proguard-rules.pro

```diff
@@ -59,3 +63,8 @@
     @com.squareup.otto.Produce public *;
 }
 
+# keep TAG class members
+-keepclassmembers class com.microsoft.embeddedsocial.ui.** {
+    public static final java.lang.String TAG;
```

Comment:
I think the spacing is different compared to previous lines

Reply:
Should be fixed with commit de97d07